CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to Docker Images Collection
+
+## Prerequisites
+
+Before submitting code contributions, you should first complete the following prerequisites.
+
+### 1. Sign the CLA
+
+Before you can contribute, we kindly ask you to sign our [Contributor License Agreement](https://cla-assistant.percona.com/percona/percona-docker) (CLA). You can do this using your GitHub account and one click.
+
+### 2. Code of Conduct
+
+Please make sure to read and observe the [Contribution Policy](code-of-conduct.md).
+
+## Submitting a pull request
+
+Improvement and bugfix tasks for Percona's projects are tracked in Jira. If you find a bug in Percona Docker Images or one of the related projects, you should submit a report to that project's [Jira issue tracker](https://jira.percona.com).
+
```

Comment:
As I can see this section is called `Submitting a pull request` but you are talking about creating jira issue. Maybe you should tell that firstly you should create ticket and submit PR and connect it with that ticket